### ISSUE TITLE
Is there an SDK that can provide a semicircular setting angle? 

### ISSUE THREAD

**Issue by lover212045** [milosmns/circular-slider-android#22]
Hi,i love this code,Is there an SDK that can provide a semicircular setting angle?

**Comment by milosmns** [comment]
Nothing I can think of right now. You can check this answer: https://github.com/milosmns/circular-slider-android/issues/9
Maybe it's something you can use and manipulate for your case

**Comment by lover212045** [comment]
> 
> 
> Nothing I can think of right now. You can check this answer: #9
> Maybe it's something you can use and manipulate for your case

No. I mean, your circle is 360 degrees. Is there a function to change it to a 180 degree semicircle

**Comment by milosmns** [comment]
Ah, that... no, not really... sorry. 

An open line would make it a curved seekbar, sort of... and unfortunately I'm quite busy with other stuff so don't have time to do improvements on this one - but I invite you to create a PR with changes if you want, and I can review it soon 🙂 

Maybe you can also check out Marcin's arc seekbar as well? Seems similar in design and functionality https://github.com/MarcinMoskala/ArcSeekBar

**Comment by lover212045** [comment]
> 
> 
> Ah, that... no, not really... sorry.
> 
> An open line would make it a curved seekbar, sort of... and unfortunately I'm quite busy with other stuff so don't have time to do improvements on this one - but I invite you to create a PR with changes if you want, and I can review it soon 🙂
> 
> Maybe you can also check out Marcin's arc seekbar as well? Seems similar in design and functionality https://github.com/MarcinMoskala/ArcSeekBar

Okay, sorry to disturb you. Thank you very much.

**Comment by milosmns** [comment]
It's not disturbing at all 😄 

Sorry I couldn't be of more help at this time
